config/environment.js

```diff
@@ -73,7 +73,7 @@ module.exports = function(environment) {
 
   }
 
-  ENV.studyId = process.env.STUDY_ID;
+  ENV.studyId = process.env.EXPERIMENT_ID;
```

Comment:
Future generations of confused people thank you. Can you add a `DEPLOYMENT NOTES` section to your PR, calling out the fact that we will need to change the private server configs before this goes live?